CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2015-10-18
```

Comment:
Please use e.g. `_Not released yet_` here so that people browsing Github won't get confused.